ci/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM python:3.6-alpine
 
 ENV PYTHONDONTWRITEBYTECODE 1
 
-RUN apk add --no-cache build-base git gcc make libffi-dev && rm -rf /var/cache/apk/*
+RUN apk add --no-cache build-base git gcc make libffi-dev openssl-dev && rm -rf /var/cache/apk/*
```

Comment:
Is this change related to the rest of the PR?

Reply:
yes, replied in a comment